deploy/charts/hcloud-csi/Chart.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: csi-hcloud
+description: Helm Chart for the CSI Driver on Hetzner Cloud
+type: application
+home: https://github.com/syself/charts/tree/main/charts/csi-hcloud
+maintainers:
+  - name: Syself
+    email: info@syself.com
+    url: https://github.com/syself
+version: 1.0.0
+appVersion: 2.1.0
```

```suggestion
version: 2.1.0
```

In addition to this change, could you please move the chart to `/chart/`?

The idea here is that the Helm chart is a native part of the application. Breaking changes to the chart would be a major version bump for the project as a whole.

/cc @apricote does this sound okay?